http-servers/src/main/java/com/palantir/remoting/http/server/JsonExceptionMapper.java

```diff
@@ -59,7 +60,9 @@ public final Response toResponse(T exception) {
                 error = SerializableError.of(message, exception.getClass(),
                         Arrays.asList(stackTrace));
             } else {
-                error = SerializableError.of(message, exception.getClass());
+                String errorId = UUID.randomUUID().toString();
+                log.error("error has been logged as {}", errorId, exception);
+                error = SerializableError.of(errorId, RuntimeException.class);
```

Comment:
Could we have a SerializableError constructor that doesn't take an exception class? Passing RuntimeException here seems unnecessary and even misleading.